meta/releasing.md

```diff
@@ -9,45 +9,44 @@ machinery works.
 The spec is released each calendar quarter. The target release dates are within the
 following ranges:
 
-* Q1: January 20-27 (critically, before FOSDEM).
-* Q2: May 20-27.
-* Q3: August 20-27.
+* Q1: January 20-27 (if needed before FOSDEM) or February 21-28.
```

Comment:
well I guess that means we're only a month late, so yay?

Honestly I question the value of attempting to set dates like this. Personally I would just say "we aim for each release to happen at most four months after the previous release, but they may happen sooner to deal with things like FOSDEM" and be done with it.

Reply:
There's still value in releasing every quarter instead of on an interval, though yes, the dates are increasingly becoming lies. We generally aim for the middle of the quarter.

Reply:
I've dropped the dates and fixed the intro sentence